README.md

```diff
@@ -109,10 +109,20 @@ This package only guarantees the compatibility with the [version v0.15.0 of Meil
 
 Please check the Vapor Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/VaporDemo).
 
+To test the Vapor demo you will need to go to the directory `Demos/VaporDemo`, you can either:
+
+ - Open the SwiftPM project in Xcode and press run, or
+ - Run `swift build` or `swift run` in the terminal.
+
 ### Perfect
 
 Please check the Perfect Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/PerfectDemo).
 
+To test the Perfect demo you will need to go to the directory `Demos/PerfectDemo`, you can either:
+
+ - Open the SwiftPM project in Xcode and press run, or
+ - Run `swift build` or `swift run` in the terminal.
+
```

Comment:
I'm not sure about writing the same text two times. What do you think of the following?

Also about step this `Open the SwiftPM project in Xcode and press run, or`. Do you need to open XCode with `/demos/VaporDemo`?
And about this step ` Run swift build or swift run in the terminal.` do you need to run that command inside the Demo folder ?

```
## Demo

To try out a demo you will need to go to its directory in `Demos/`. After which you can either: 

 - Open the SwiftPM project in Xcode and press run, or
 - Run `swift build` or `swift run` in the terminal.

### Vapor

Please check the Vapor Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/VaporDemo).

### Perfect 

Please check the Perfect Demo source code [here](https://github.com/meilisearch/meilisearch-swift/tree/master/Demos/PerfectDemo).

```

Reply:
I like the non-repetitive way 👍